tests/test_visual.py

```diff
@@ -262,3 +263,101 @@ def test_add_single_bbox_on_image(mock, get_image_and_bbox):
         font_size=100,
         box_line_width=15,
     )
+
+
+def create_keypoint_template(guid):
+    """prepare a fake template"""
+
+    templates = [
+        {
+            "template_id": guid,
+            "template_name": "test_template",
+            "key_points": [
+                {
+                    "label": "lower_left",
+                    "index": 0,
+                    "color": {"r": 1, "g": 0, "b": 0, "a": 1},
+                },
+                {
+                    "label": "upper_left",
+                    "index": 1,
+                    "color": {"r": 1, "g": 0, "b": 0, "a": 1},
+                },
+                {
+                    "label": "upper_right",
+                    "index": 2,
+                    "color": {"r": 1, "g": 0, "b": 0, "a": 1},
+                },
+                {
+                    "label": "lower_right",
+                    "index": 3,
+                    "color": {"r": 1, "g": 0, "b": 0, "a": 1},
+                },
+            ],
+            "skeleton": [
+                {
+                    "joint1": 0,
+                    "joint2": 1,
+                    "color": {"r": 0, "g": 1, "b": 0, "a": 1},
+                },
+                {
+                    "joint1": 1,
+                    "joint2": 2,
+                    "color": {"r": 0, "g": 1, "b": 0, "a": 1},
+                },
+                {
+                    "joint1": 2,
+                    "joint2": 3,
+                    "color": {"r": 0, "g": 1, "b": 0, "a": 1},
+                },
+                {
+                    "joint1": 3,
+                    "joint2": 0,
+                    "color": {"r": 0, "g": 1, "b": 0, "a": 1},
+                },
+            ],
+        }
+    ]
+
+    return templates
+
+
+def create_keypoints(guid, x, y, dim):
```

Comment:
Can this be declared as pytest fixture using `@pytest.fixture` decorator?

Reply:
fixed